doc/man3/OSSL_PROVIDER.pod

```diff
@@ -140,6 +144,11 @@ OSSL_PROVIDER_get_capabilities() return 1 on success, or 0 on error.
 OSSL_PROVIDER_load() and OSSL_PROVIDER_try_load() return a pointer to a
 provider object on success, or B<NULL> on error.
 
+OSSL_PROVIDER_do_all() returns 1 if the callback I<cb> returns 1 for every
+provider it is called with, or 0 if any provider callback invocation returns 0;
+callback processing stops if the first callback invocation on a provider
+returns 0.
```

Comment:
I know now what you're trying to say, but I think it doesn't quite sound right. How about:

"callback processing stops at the first callback invocation on a provider that returns 0"

Reply:
Thanks. So inserted.